src/commands/create/spec/create.spec.server.ts

```diff
@@ -22,6 +23,7 @@ describe('sasjs create', () => {
     const appName = `test-app-create-.-${generateTimestamp()}`
 
     process.projectDir = path.join(__dirname, appName)
+    await setConstants()
```

Comment:
should be moved to `beforeEach`

Reply:
These are in right place, `process.sasjsConstants` are dependent on `process.projectDir`